lib/slack-ruby-bot/hooks/hello.rb

```diff
@@ -1,9 +1,13 @@
 module SlackRubyBot
   module Hooks
-    module Hello
-      extend Base
+    class Hello
+      attr_accessor :logger
 
-      def hello(client, _data)
+      def initialize(logger)
```

Comment:
The argument should be `options`, a Hash, so we can evolve this in the future without changing the interface.